win-spout.cpp

```diff
@@ -307,25 +372,38 @@ static void win_spout_render(void *data, gs_effect_t *effect)
 	struct win_spout *context = (win_spout *)data;
 
 	if (!context->active) {
-		debug("inactive");
+		if (context->render_status != -1) {
+			debug("inactive");
+			context->render_status = -1;
+		}
 		return;
 	}
 
 	// tried to initialise again
 	// but failed, so we exit
 	if (!context->initialized) {
-		debug("uninit'd");
+		if (context->render_status != -2) {
+			debug("uninit'd");
+			context->render_status = -2;
+		}
 		return;
 	}
 
 	if (!context->texture) {
-		debug("no texture");
+		if (context->render_status != -3) {
+			debug("no texture");
+			context->render_status = -3;
+		}
 		return;
 	}
 
-	info("rendering context->texture");
+	if (context->render_status != 0) {
+		info("rendering context->texture");
+		context->render_status = 0;
+	}
+	
 
-	effect = obs_get_base_effect(OBS_EFFECT_OPAQUE);
+	effect = obs_get_base_effect(OBS_EFFECT_PREMULTIPLIED_ALPHA);
```

Comment:
This might solve #4

Reply:
I am thinking whether i should give a choose to user or not . Because i notice that some softwares didn't have correct alpha channel output and it will cause these software looks weird.

Reply:
Yes I think that's a good idea. This is the main thing I wanted to test first.